charts/seed-controlplane/charts/kube-controller-manager/values.yaml

```diff
@@ -22,3 +22,4 @@ resources:
 horizontalPodAutoscaler:
   downscaleDelay: 15m
   upscaleDelay: 1m
+  tolerance: 0.2
```

Comment:
Perhaps we can keep the default `tolerance` in the values.yaml to `0.1` and set it to `0.2` only for seeds here -> https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/controlplane.go#L316-L319?

Reply:
Also, do we need to tweak CoreDNS' limits because now `tolerance` is `0.2`?

Reply:
Yes, will keep the tolerance for shoot to 0.1
Thanks for suggestion.

For CoreDNS:
Current requested: 100m, Limit: 200m
with 0.2 tolerance and 80% threshold, coreDNS will be scaled at 96m CPU usage.
So tweaking limits should not be required. WDYT?

Reply:
Cool. Thanks!